### PR TITLE
feat(ci): Setup unit-tests and linting steps on Jenkins pipeline

### PR DESCRIPTION
**Pull request by ganoze**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -29,6 +29,22 @@ stage('Source') {
...
 
 try {
+  stage('Unit tests') {
+    node {
+      sh 'setup_centreon_build.sh'
+      sh "./centreon-build/jobs/ui/ui-unittest.sh"
+      junit 'ut.xml'
+      recordIssues(
+        enabledForFailure: true,
+        tools: [tsLint(pattern: 'codestyle.xml')],
```
Comment (bdauria) [inline Jenkinsfile]: This should use ESLint instead of TSLint

Comment (ganoze) [reply Jenkinsfile]: Good catch ! Fixed.

**Comment by bdauria** [comment]
@ganoze Can you setup the automatic check on Jenkins?

**Comment by ganoze** [comment]
@bdauria Done.

**Pull request merged**